### PR TITLE
Fix visualization of pressure when 0mBar

### LINKED ISSUE #1744: Mission header looks bad when pressure is 0
**Describe the bug**
See screenshot

**To Reproduce**
* Run mission on robot where pressure is 0

**Expected behavior**
Should look like the second screenshot

**Screenshots**
When pressure is 0
Skjermbilde 2024-09-20 kl  10 02 17

When pressure is something else than 0
image

### PR DESCRIPTION
**Pull request by Eddasol**
Closes #1744

### CODE REVIEW

**Comment by Eddasol** [comment]
Before:
image
Now:
image

File: frontend/src/components/Displays/ConfirmScheduleDialogs/ConfirmScheduleDialog.tsx [hunk 1]
```diff
@@ -23,17 +23,25 @@ export const ScheduleMissionWithConfirmDialogs = ({
     const { enabledRobots } = useRobotContext()
     const [robot, setRobot] = useState<Robot>()
 
-    const isBatteryInsufficient = (currentRobot: Robot) =>
-        currentRobot.batteryLevel &&
-        currentRobot.model.batteryWarningThreshold &&
-        currentRobot.batteryLevel < currentRobot.model.batteryWarningThreshold
+    const isBatteryInsufficient = (currentRobot: Robot) => {
+        const has_battery_value = currentRobot.batteryLevel !== null && currentRobot.batteryLevel !== undefined
```
Comment (mrica-equinor) [inline frontend/src/components/Displays/ConfirmScheduleDialogs/ConfirmScheduleDialog.tsx:27]: I would day the has_battery is kinda of a python thing and in typescript we would normally use hasBatteryValue? maybe change the variables names

File: frontend/src/components/Displays/ConfirmScheduleDialogs/ConfirmScheduleDialog.tsx [hunk 2]
```diff
@@ -23,17 +23,25 @@ export const ScheduleMissionWithConfirmDialogs = ({
...
+        return (
+            has_battery_value &&
```
Comment (mrica-equinor) [inline frontend/src/components/Displays/ConfirmScheduleDialogs/ConfirmScheduleDialog.tsx:29]: same thing with this variable name

**Review by mrica-equinor — APPROVED** [review]
LGTM :)

**Pull request merged**